src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="#/" method="POST"  data-qa="big">
```

Comment:
@MabuzaM it looks like tests can't find for both of your forms since you haven't explicitly specified their measures. (Error says `Node has 0 height`)
Try to style both forms with a class, that will define `width` and `height`

Reply:
Okay Thank you @meforoma. Have done so but now it won't run UI test at all as it fails to launch the browser.

Reply:
How is my search bar now? 
Am having trouble fixing the font

Reply:
Had to set the body margin to 0 for the test to pass